tests/Data/ProgrammesDb/EntityRepository/PromotionRepository/FindPromotionsByPidTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @coversDefaultClass BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\PromotionRepository
```

Comment:
This test doesn't end up inadvertently covering large amounts of other stuff, so there is no need to manually restrict it to just this one class.